src/server/server.ts

```diff
@@ -366,6 +366,9 @@ export default class TONQServer {
         const typeDefs = endPoint.typeDefFileNames
             .map(x => fs.readFileSync(path.join("res", x), "utf-8"))
             .join("\n");
+
+        const subscrWSSet = new Set(); 
```

Comment:
I think we need it :-(,
We need to close all WS subscriptions (not ALL ws!! connections because now ws is being used by Queries), so we need to keep references to all open WS subscriptions and I haven't found another way to keep track of them.